python/idsse_common/idsse/common/log_util.py

```diff
@@ -137,11 +137,20 @@ def get_default_log_config(level: str, with_corr_id=True):
                 'formatter': 'standard',
                 'filters': filter_list,
             },
+            'rabbit': {
+                'class': 'python_logging_rabbitmq.RabbitMQHandler',
+                'host': 'localhost',
+                'port': 5672,
+                'formatter': 'standard',
+                'filters': filter_list,
+                'level': 'ERROR',
```

Comment:
I wonder about this being hard coded. Probably fine for now but might want to address in a future log_utils cleanup.

Reply:
I guess we could parameterize the function for these elements and set the default values.